src/MergeStream.js

```diff
@@ -41,7 +41,7 @@ class MergeStream {
 
     // Set up the on end handler for the sourceStream
     inputStream.on("end", () => {
-      console.log(`>>> Unmerged ${inputStream.name}`);
+      console.log(`>>> Merged ${inputStream.name}`);
```

Comment:
In the future, we should create a logging method that can be set with a flag whether it outputs or not.